flocx_market/tests/unit/models/test_offer_unit.py

```diff
@@ -0,0 +1,25 @@
+from flocx_market.tests.unit.unit_base_test import UnitBaseTest
```

Comment:
Please take this out of the pull request.  This pull request is for implementing the Bid api, and should not have any offer related funcions.